tox.ini

```
@@ -7,6 +7,8 @@ minversion = 2.4
 envlist = default
 
 [testenv]
+deps=
+    -r{toxinidir}/requirements.txt
```

Looks like we can maybe remove `{toxinidir}`? https://tox.readthedocs.io/en/latest/example/basic.html#depending-on-requirements-txt-or-defining-constraints
```suggestion
    -rrequirements.txt
```